### PR TITLE
feat(generic-metrics): Add gauges to docker compose, re-try

### PR DESCRIPTION
**Pull request by ayirr7**
Adding this missing consumer, this is a repeat of https://github.com/getsentry/self-hosted/pull/2757

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -293,6 +293,9 @@ services:
   snuba-generic-metrics-counters-consumer:
     <<: *snuba_defaults
     command: rust-consumer --storage generic_metrics_counters_raw --consumer-group snuba-gen-metrics-counters-consumers --auto-offset-reset=latest --max-batch-time-ms 750 --no-strict-offset-reset
+  snuba-generic-metrics-gauges-consumer:
+    <<: *snuba_defaults
+    command: consumer --storage generic_metrics_gauges_raw --consumer-group snuba-gen-metrics-gauges-consumers --auto-offset-reset=latest --max-batch-time-ms 750 --no-strict-offset-reset
```
Comment (aldy505) [inline docker-compose.yml:298]: Are you sure that you don't want to use rust-consumer?

**Review by aldy505 — APPROVED** [review]

**Comment by hubertdeng123** [comment]
Wouldn't this also require us to flip the feature flags for custom metrics? Otherwise, what is this consumer used for?

File: docker-compose.yml (hunk 1 above)
Comment (ayirr7) [reply docker-compose.yml:298]: Sorry, let me fix that.

**Comment by ayirr7** [comment]
> Wouldn't this also require us to flip the feature flags for custom metrics? 

Could you point me to the feature flags you're referring to? But yes, you are correct that today the gauges consumer is only used by the custom metrics functionality.

**Comment by aldy505** [comment]
> Wouldn't this also require us to flip the feature flags for custom metrics? Otherwise, what is this consumer used for?

I think it's fine since AFAIK Metrics isn't GA yet. We can add the flags in a separate PR.

**Comment by aldy505** [comment]
Ping @ayirr7. We are 5 more days until the next release.

**Comment by patrickklaeren** [comment]
Is there any news on when this might be merged?

I understand this has now missed the window for .7

**Comment by ayirr7** [comment]
> I believe adding the gauge consumer now is a good idea, since it will ingest some data beforehand, that can be viewed later on the UI when GA's arrived -- so the Metrics UI won't be empty when it's enabled.

I believe this should be the case here. The consumer ingest any relevant data. Once any feature(s) are enabled, they can be used to query the data.

**Review by hubertdeng123 — APPROVED** [review]

**Comment by aldy505** [comment]
Thanks Riya! @ayirr7

**Pull request merged**